src/commands.ts

```diff
@@ -13,8 +13,16 @@ import { type MaybeArray } from "./utils.ts";
 
 export type SetMyCommandsParams = {
     /**
-     * Scope
+     * Interface for grouping {@link BotCommand}'s that might or might not be related to each other
+     * by scope and/or language.
+     *
+     * If @param scope it's defined, they will only take effect on that scope.
+     * If @param language_code it's defined, they will only take effect on users of that language.
```

Comment:
These should go before each property, without the `@param` tag, because they describe the property itself.